index.js

```diff
@@ -42,12 +42,13 @@ co.wrap = function (fn) {
 
 function co(gen) {
   var ctx = this;
+  var args = slice.call(arguments, 1)
```

Comment:
IMO that optimization should be done separately in another PR after this is merged.